async-queue/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "async-queue"
```

Comment:
We'll need to rename this crate since `async-queue` was taken by someone else recently.